javascript-algorithms-and-data-structures/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/index.js

```diff
@@ -1,19 +1,13 @@
-const postsContainer = document.getElementById("posts-container");
+const FORUM_LATEST = "https://forum-proxy.freecodecamp.rocks/latest";
```

Comment:
I think it's fine to use screaming snake case for these constant variables, though I do have a slight preference for regular old camel case.

Either way works, though if we stick with screaming snake case, I believe this is the first project to use it, and we'll need to explain why this format for naming variables is used.